index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <nav>
+        <h1>Dr. Borlaug
+        </h1>
+        <h1>The man who saved a billion lives</h1>
+    </nav>
+
+    <div class="background-image">
+        <img src="images/norman_borlaug_and_wheat_breeding_trainees_1972_1.jpg" alt="">
+        <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+    </div>
+
+    <div class="biography">
+        <h3>Here's a time line of Dr. Borlaug's life:</h3>
+        <ul>
+            <li> <b>&#8226 1914</b> - Born in Cresco, Iowa</li><br>
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+            
+            <li> <b>&#8226 1914</b> - Born in Cresco, Iowa</li><br>
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+            
+            <li> <b>&#8226 1914</b> - Born in Iowa, Cresco, </li><br>
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+            <li> <b>&#8226 1914</b> - Born in Cresco, Iowa</li><br>
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
+            
+            
+            
+            
+            
+        </ul>
```

Comment:
please fill the gap from line 47 to 51 .

Reply:
- [ ] kindly give a reason (WHY) for the changes requested

Reply:
Done